tests/test_calculator.py

```diff
@@ -0,0 +1,7 @@
+import pyhf
+import pyhf.infer.calculators
+
+
+def test_calc_dist():
+    asymptotic_dist = pyhf.infer.calculators.AsymptoticTestStatDistribution(0.0)
+    assert asymptotic_dist.pvalue(1) == asymptotic_dist.cdf(-1)
```

Comment:
why is this true? If AsymptoticTestStatDistribution has `0.0 = -u^/sigma` with variance 1.0, then...>?